cmd/version.go

```diff
@@ -8,26 +8,38 @@ import (
 )
 
 func versionCmd() *cobra.Command {
-	return &cobra.Command{
+	var silent bool
+
+	command := cobra.Command{
 		Use:   "version",
 		Short: "Print version/build info",
 		Long:  "Print version/build information",
 		Run: func(cmd *cobra.Command, args []string) {
-			printVersion()
+			printVersion(silent)
 		},
 	}
+
+	command.PersistentFlags().BoolVarP(&silent, "short", "s", false, "Simplified print version for resumed printing")
+
+	return &command
 }
 
-func printVersion() {
+func printVersion(silent bool) {
 	const secFmt = "%-10s "
+	var outputColor color.Paint
 
-	printLogo(color.Cyan)
-	printTuple(secFmt, "Version", version)
-	printTuple(secFmt, "Commit", commit)
-	printTuple(secFmt, "Date", date)
+	if silent {
+		outputColor = color.White
+	} else {
+		outputColor = color.Cyan
+		printLogo(outputColor)
+	}
+	printTuple(secFmt, "Version", version, outputColor)
+	printTuple(secFmt, "Commit", commit, outputColor)
+	printTuple(secFmt, "Date", date, outputColor)
 }
 
-func printTuple(format, section, value string) {
-	fmt.Printf(color.Colorize(fmt.Sprintf(format, section+":"), color.Cyan))
+func printTuple(format, section, value string, outputColor color.Paint) {
```

Comment:
@brunohms I think the idea here is to strip out all ansi color tags since folks just need to grep thru the output and extract the version info. Thus when in short mode we should just print to stdout and remove the colorizer since it will add extra ansi color chars to the output which defies the purpose of the short version. So we could take a color pointer and check if the ptr is nil we can just dump the string to stdout or (better) define a color.Paint=none and make the change on the colorizer.

Reply:
Changed it on my other commit.
See the other discussion for more details.